content/dstu2/document-reference.md

```diff
@@ -13,7 +13,7 @@ title: DocumentReference | DSTU 2 API
 
 ## Create
 
-Create new documents. Only documents that would be classified as notes are currently supported.
+Create new documents. Currently only XHTML formatted documents are supported. Only documents that need to be associated to the patient's chart should be written. Please validate your document is valid XHTML using any available validator like the one linked [here]. 
```

Comment:
Maybe add this as a "note" similar with what we've done elsewhere?

We should also add it to the documentation for content.attachment.data - it's base64 encoded XHTML.